doc/src/docs/InputOutputReference/01d-InputOutputReference.md

```diff
@@ -15785,6 +15787,18 @@ The name of the supply fan outlet node.
 
 The name of a NodeList object containing the names of the HVAC system nodes or the HVAC System Node Name where temperature setpoints will be established by this setpoint manager.
 
+#### Field: Cooling Coil Inlet Node Name
+
+The name of the cooling coil inlet node.
+
+#### Field: Cooling Coil Outlet Node Name
+
+The name of the cooling coil outlet node.
+
+#### Field: Minimum Temperature at Cooling Coil Outlet Node
+
+In order to prevent cooling coil freezing, the minimum temperature at the cooling coil outlet is required based on requirement of ASHRAE Standard 90.1. The default value is 7.2C.
```

Comment:
@lgu1234 Given how large and complicated Controller:OutdoorAir has become, it might be useful to split off the economizer portions in to a separate object.  Something to consider if we reorganize the OA controls.